framework/include/base/FEProblemBase.h

```diff
@@ -665,6 +666,13 @@ class FEProblemBase :
   MooseSharedPointer<MultiApp> getMultiApp(const std::string & multi_app_name);
 
   /**
+   * Execute MultiAppTransfers associate with execution flag and direction.
+   * @param type The execution flag to execute.
+   * @param direction The direction (to or from) to transfer.
+   */
+  void execMultiAppTransfers(ExecFlagType type, MultiAppTransfer::DIRECTION direction);
```

Comment:
Why do we have this new method? We already have `execTransfers()` which should handle this case just fine.